FirstClasses/fizzbuzz/src/fizzbuzz.erl

```diff
@@ -3,4 +3,21 @@
 -export([fizzbuzz/0]).
 
 fizzbuzz() ->
-    ok.
+    lists:map(
+    fun(Number) ->
+        case Number of
```

Comment:
Well done 👍 You might consider extracting the fun to a named function like this:
```erlang
fizzbuzz() -> 
   lists:map(fun  fb/1, lists:seq(1, 100)).

fb(Number) when Number rem 3 == 0 andalso Number rem 5 ==0 -> fizzbuzz;
fb(Number) when Number rem 3 == 0 -> fizz;
fb(Number) when Number rem 5 == 0 -> buzz;
fb(Number) -> Number. 
```